test/testother.cpp

```diff
@@ -6447,6 +6449,22 @@ class TestOther : public TestFixture {
                         ));
         ASSERT_EQUALS("", errout.str());
     }
+
+    void funcDefArgNoName() {
+        /* Caused a false positive:
+         * [test.cpp:2]: (performance) Function parameter 'DS' should be passed by reference.
+         * While there is no problem at all
+         */
+        check("static const std::string DS = \"abcd\";\n"
+              "static void dsth(const std::string & = DS);\n"
+              "void dsth(const std::string &s) {}\n"
+              "int main(void) {\n"
```

Comment:
why do you have this main function? I assume the false positive is written without it.

Reply:
Avoid the unused function error.
But not 100% sure, if this is necessary for the test classes.

The test itself is a copy from a local file which compiled, but I can remove the main function, if it is not necessary in here